front/pages/security.tsx

```diff
@@ -44,46 +45,56 @@ export const getServerSideProps = makeGetServerSidePropsRequirementsWrapper({
   }
 
   return {
-    props: { gaTrackingId: config.getGaTrackingId(), shape: 10 },
+    props: {
+      gaTrackingId: config.getGaTrackingId(),
+      shape: getParticleShapeIndexByName(shapeNames.icosahedron),
+    },
   };
 });
 
 export default function Security() {
   return (
     <>
       <HeaderContentBlock
-        title={
-          <>
-            For enterprise&nbsp;AI,
-            <br />
-            security is&nbsp;non-negotiable
-          </>
-        }
+        title={<>Security is&nbsp;non-negotiable</>}
```

Comment:
Any reason to keep title being a `ReactNode` over a simple string?

Reply:
Mostly, we're going to do a new pass on all of them with stuff like <br/> and such.
I'll clean then, the purpose of merging is to stop squating front-edge as much as possible.